### PR TITLE
Hotfix/images

### PR DESCRIPTION
**Pull request by abea**
Fixes usages of the image macro.

### CODE REVIEW

File: lib/modules/articles-featured-widgets/views/widget.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
   <div class="o-container o-container--medium c-article-featured-widget__container">
     <div class="c-article-featured-widget__media">
       <a href="{{ w._article._url }}">
-        {{ image.render(w._article.thumbnail.items[0]._pieces[0]) }}
+        {% set thumbnail = apos.images.first(w._article.thumbnail.items) %}
```
Comment (grdunn) [inline lib/modules/articles-featured-widgets/views/widget.html]: Can you just pass `w._article.thumbnail`?

File: lib/modules/articles-featured-widgets/views/widget.html [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
+        {{ image.render(thumbnail) }}
```
Comment (grdunn) [inline lib/modules/articles-featured-widgets/views/widget.html:10]: Do we need to check if `thumbnail` exists before passing it to `image.render()` or will it check for it inside the macro?

File: lib/modules/articles-featured-widgets/views/widget.html (hunk 1 above)
Comment (boutell) [reply lib/modules/articles-featured-widgets/views/widget.html]: You can, it'll dig.

File: lib/modules/articles-featured-widgets/views/widget.html (hunk 2 above)
Comment (abea) [reply lib/modules/articles-featured-widgets/views/widget.html:10]: I'll fall back to a placeholder image.

Comment (grdunn) [reply lib/modules/articles-featured-widgets/views/widget.html:10]: 👍

File: lib/modules/articles-featured-widgets/views/widget.html (hunk 1 above)
Comment (grdunn) [reply lib/modules/articles-featured-widgets/views/widget.html]: I think in most Punk Ave projects we pass the image object and let the helper dig down for the image. Might be good for consistency's sake (and less words).

Comment (abea) [reply lib/modules/articles-featured-widgets/views/widget.html]: Updated.

Comment (grdunn) [reply lib/modules/articles-featured-widgets/views/widget.html]: save the trees.

**Review by grdunn — APPROVED** [review]

**Pull request merged**